Source/Core/Common/Common.vcxproj

```diff
@@ -175,6 +176,7 @@
     <ClCompile Include="Hash.cpp" />
     <ClCompile Include="IniFile.cpp" />
     <ClCompile Include="JitRegister.cpp" />
+    <ClCompile Include="LdrWatcher.cpp" />
```

Comment:
`git grep` shows cmake did not even have ucrtFreadWorkaround ...I'd like if we could stop pretending cmake/windows is actually a thing.